daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionCoderSpec.scala

```diff
@@ -789,7 +841,16 @@ class TransactionCoderSpec
   private[this] def normalizeExe[Nid](exe: Node.NodeExercises[Nid, ContractId]) =
     exe.copy(
       chosenValue = normalize(exe.chosenValue, exe.version),
-      exerciseResult = exe.exerciseResult.map(normalize(_, exe.version)),
+      exerciseResult = exe.exerciseResult match {
+        case None =>
+          if (exe.version >= minExceptions) {
```

Comment:
What do we need this normalization for? The generator already takes the version into account so I don’t get when we will end up with one with a version that doesn’t support exceptions and this being `None` (outside of the tests where we disable the version check but those shouldn’t normalize afaict).

Reply:
This is necessary because various existing tests call `updateVersion` after generation. For example `"do NodeExercises"`